charts/rancher-sriov/sriov/0.1.1+up0.1.0/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
Assets need to be regenerated. `0.1.1` is incorrect, right?

Reply:
The upstream version is 0.1.0 --> https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/deployment/sriov-network-operator/Chart.yaml#L3. Not sure how is the last 1 filled, any idea?

Reply:
Gotcha. So the fix is to the CRD chart version, not to this chart. Will leave a comment for that in my next review, but resolving this.